app/Resources/views/themes/tastefulplanet/full/product_list.html.twig

```diff
@@ -0,0 +1,162 @@
+{% extends "@ezdesign/pagelayout.html.twig" %}
+
+{% block page_head %}
+    {% set title = ez_content_name(content) ~ " - " ~ content.contentInfo.publishedDate|localizeddate('short', 'short', app.request.locale) %}
```

Comment:
This is a copy/paste from other full views. Remove the date. There is no need here... and we'll have NovaSEOBundle ;)